packages/react-components/src/organisms/__tests__/CalendarList.test.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import CalendarList from '../CalendarList';
+
+it('Renders calender list', () => {
+  const { getByRole } = render(<CalendarList calendars={[]} />);
+  expect(getByRole('heading').textContent).toMatch(/subscribe/i);
+});
+
+it('Renders calender list item with colour', () => {
+  const { getByText } = render(
+    <CalendarList
+      calendars={[
+        {
+          colour: 'red',
+          name: 'Test Event',
+          id: '1',
+        },
+      ]}
+    />,
+  );
+  expect(getComputedStyle(getByText('●')).color).toMatch('red');
+  expect(getByText('Test Event')).toBeVisible();
+});
+
+it('Correctly generates the subscribe link', () => {
+  const { getByRole } = render(
+    <CalendarList
+      calendars={[
+        {
+          colour: 'red',
+          name: 'Test Event',
+          id: '1',
+        },
+      ]}
+    />,
+  );
+  expect(getByRole('link')).toHaveAttribute(
+    'href',
+    'https://calendar.google.com/calendar/u/1/r?cid=1',
```

Comment:
`/u/1/`? o.O

Reply:
I have no idea. I just used the example from the trello card. (https://trello.com/c/GvMZ2DV4/832-create-a-component-with-the-calendar-list). If we had some example data from somewhere that would be helpful

Reply:
Yeah I'm sure that wasn't there on purpose. Let's not assume people want to add it to the second google account. In fact, it is allowed to just omit that part entirely: `.../calendar/r?cid=...`, that would be preferable I think

Reply:
The `/1` is an hint for google to know what logged in user has access to the calendar. `/0` is the first user, `/1` the second, ...

Reply:
Yeah I know what it does, but the second user seems arbitrary, and omitting it just leaves it up to Google to decide, presumably always meaning the first user